utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/netstat.lua

```diff
@@ -1,10 +1,20 @@
+local netsubstat = {
+    "IcmpMsg",
+    "Icmp",
+    "IpExt",
+    "Ip",
+    "TcpExt",
+    "Tcp",
+    "UdpLite",
+    "Udp"
+}
+
```

Comment:
@champtar should be better not to redefine the array on every scrape, right?

Reply:
yup